### PR TITLE
Add GitHub Pages deployment instructions

### PR DESCRIPTION
**Pull request by christopher-besch**

### CODE REVIEW

File: docs/source/use/export_presentation.rst [hunk 1]
```diff
@@ -28,6 +28,54 @@ Instead of running a local web server you can host the project on an online web
 That way you can access it from anywhere.
 This is how the `example <https://manimcommunity.github.io/manim_editor/Tutorial/index.html>`__ has been created.
 
+If you intend to use GitHub Pages, you have to create a GitHub repository first.
+All the steps you need to follow are explained in [this official GitHub Pages tutorial](https://docs.github.com/en/pages/getting-started-with-github-pages/creating-a-github-pages-site).
+The tutorial diverges at one point:
+You don't have to create any markdown or HTML files.
+Instead you have to copy the contents of the project folder (after you've exported the presenter) into the root of the repo.
```
Comment (behackl) [inline docs/source/use/export_presentation.rst:35]: the paragraph reads a bit less clunky IMO, but that might be just my opinion.

```suggestion
Instead, copy the contents of the project folder (after you've exported the presenter) into the root of the repo.
```

**Review by behackl — APPROVED** [review]
Read the update, looked at image, LGTM!

**Comment by christopher-besch** [comment]
Funny how neither one of us noticed some astray Markdown in a reStructuredText.
image

**Comment by behackl** [comment]
> Funny how neither one of us noticed some astray Markdown in a reStructuredText.
> image

Ha! I've been writing too much markdown for the new FAQ section recently :-)

**Pull request merged**